mkposters/post_install.py

```diff
@@ -0,0 +1,53 @@
+import subprocess
+
+
+def main():
+    bash_cmd = """
+    # Detect the compute architecture (linux-arm64, macos-arm64, macos-x64) and download the appropriate distribution release of dart-sass
+    # https://github.com/sass/dart-sass/releases/tag/1.50.1
+
+    sass_release="1.50.1"
+    arch=$(uname -m)
+    kernel=$(uname -s)
+
+    echo Detected architecture: $arch and kernel: $kernel.
+
+    if [[ "$arch" =~ ^(arm64|aarch64)$ ]] && [ "$kernel" = "Linux" ]; then
+        curl -sL "https://github.com/sass/dart-sass/releases/download/${sass_release}/dart-sass-${sass_release}-linux-arm64.tar.gz" > dart.tar.gz && tar -xzf dart.tar.gz && rm dart.tar.gz
+    elif [ "$arch" = "x86_64" ] && [ "$kernel" = "Linux" ]; then
+        curl -sL "https://github.com/sass/dart-sass/releases/download/${sass_release}/dart-sass-${sass_release}-linux-x64.tar.gz" > dart.tar.gz && tar -xzf dart.tar.gz && rm dart.tar.gz
+    elif [ "$arch" = "arm64" ] && [ "$kernel" = "Darwin" ]; then
+        curl -sL "https://github.com/sass/dart-sass/releases/download/${sass_release}/dart-sass-${sass_release}-macos-arm64.tar.gz" > dart.tar.gz && tar -xzf dart.tar.gz && rm dart.tar.gz
+    elif [ "$arch" = "x86_64" ] && [ "$kernel" = "Darwin" ]; then
+        curl -sL "https://github.com/sass/dart-sass/releases/download/${sass_release}/dart-sass-${sass_release}-macos-x64.tar.gz" > dart.tar.gz && tar -xzf dart.tar.gz && rm dart.tar.gz
```

Comment:
If a file called `dart.tar.gz` already exists in thr CWD then this will overwrite and remove it. Better to either place it directly in the target directory (which we control) or to download it to a temporary directory.

Reply:
Fixing in commit.